docs/pages/lookback_zscore.md

```diff
@@ -0,0 +1,94 @@
+# The "lookback z-score method" of benchmark result analysis
+
+A core value of Conbench is the ability to compare incoming benchmark results with historic results to make data-driven decisions about performance.
+The primary method of doing this in Conbench is the "lookback z-score" analysis.
+This page describes how that process works.
+
+## Overview
+
+The lookback z-score analysis assigns a numeric z-score value to a benchmark result that represents how "good" the result is compared to a historic distribution of similar results.
+Coupled with a threshold, this can be used to determine whether the benchmark result is indicative of a performance regression or improvement.
```

Comment:
```suggestion
Coupled with a threshold, this can be used to determine whether the benchmark result is indicative of a performance regression or improvement.
For example, if a result is very far from the historical distribution and takes longer, that is indicative of a performance regression.
```